src/main/java/seedu/address/logic/parser/profile/ProfileCommandParser.java

```diff
@@ -19,7 +19,7 @@
  */
 public class ProfileCommandParser implements Parser<ProfileCommand> {
     private static final Pattern PROFILE_COMMAND_FORMAT = Pattern.compile(
-            "\\s+(-(?<profileOption>\\S*))(\\s+(?!-)\\S+)*");
+            "\\s+(-(?<profileOption>\\S*))(\\s+(?!-)\\S+)*\\s*");
```

Comment:
I believe this change is not necessary because the `userInput` is trimmed in `NuSchedulerParser`.